### ISSUE TITLE
The front needs more love!

### ISSUE THREAD

**Issue by lakruzz** [Praqma/code-conf.com#100]
The front page of code-conf.com needs some love.

- [ ] After we've introduced "layered" events (academy -> 4 sub pages) it stats to get messy
- [ ] Its not clear what we're listing here (same issue raised in #88 )

---

We could need some input on what would work @jonnycoull : maybe you have some ideas?

![image](https://cloud.githubusercontent.com/assets/155492/14581959/1a661574-03fb-11e6-85ad-35d8c3aafb8a.png)

**Comment by jonnycoull** [comment]
Not sure having a great long list as a home page makes for a welcoming website. if I refer you back to the original. it ws more a cherrypicking exercise of 4-5 main events, we can't shout equally about everything. its impossible. and have a sub list down one side of them all. also having all the colours on the homepage is messy.

**Comment by jonnycoull** [comment]
like this? @lakruzz @illus0r 
![screen shot 2016-04-19 at 15 44 44](https://cloud.githubusercontent.com/assets/17589853/14643273/b43e5b92-0645-11e6-9836-3d526fe461fb.png)

**Comment by illus0r** [comment]
@jonnycoull good job!

1. Are these events meetups or hackathones or trainings? Not clear now
2. If you'd use real data in mockups, we could notify that some events will be duplicated in right column.
3. This thing looks like a menu ![image](https://cloud.githubusercontent.com/assets/797993/14672603/9130e8e2-0701-11e6-8a75-8422d7541ac0.png)
1. Where can I find 5th and others events?
4. what's a new color coding? That's not clear because of lack of real data.
1. What's a color for links?

**Comment by jonnycoull** [comment]
Can you send me links to the page as is that I can view? I need to get a
handle on the volume of items (events, meet-ups etc) were dealing with as
its vastly expanded since our initial meetup.

answers..

1. The panel would be a mixed bag. so everything. just the first 2 months
for exe, then SEE ALL link will take you to a more intestine page
containing them all. I've an idea about that as well i need to mock up.
2. real data, for the time being its just an idea, so don't take that look
for granted it needs work.
3. it sort of is a menu, but at the same time a miniature list so you can
get the idea of volume. instead of having a huge list that will not be
visible of the main page.
4. SEE ALL BUTTON. which will do the same job as the menu items
CONFERENCES, MEETUPS, and what ever else we think up.

5. new colour coding is in reference to all the event locations. we need to
divide the max event list so we can see how many there will be one day to
make sure we don't run into another CODE ACADEMEY situation where its just
too much colour. once thats done ill let you know, keep going as you are
and once I've addressed my issues ill show you new layouts and we can have
a hangout?
6. as for links, I'm thinking again so we don't have a rainbow on our hands
can they be our code con orange on all links?

in short none of this is to implement yet, all needs a lot of design work
on my part, but if you can send links of all the stuff that live so far
that would be helpful.

thanks @illus0r

**Comment by lakruzz** [comment]
@jonnycoull good job
@illus0r thank for input

I like it!

I especially like that I only need to briefly _mention_ an issue, and you get it right away - you're clever!

I like the fact that we're making this look like a gig listing of a concert hall - maybe it's not Punk, but it _is_ Rock'n'roll!

I would like a front page along these lines - I realize that it's not done yet, but we should have the two of you work on breaking this down to something that is implementable.

@meekrosoft do you wanna chip in? What do you think!

**Comment by illus0r** [comment]
@jonnycoull 
>if you can send links of all the stuff that live so far
that would be helpful.

You can find something on [code-conf.com](http://code-conf.com). Collecting the rest is a big job. You can parse our meetup events to get more data to fill the mockup.

Sorry for too early criticising, I did know the mockup is unfinished yet

**Comment by jonnycoull** [comment]
So for graphics, placement (not actual word content I've spoken to @adamhenriques  about that) here's a draft of the site landing page. @illus0r @lakruzz @meekrosoft 

• full list on menu bar at top. 
• new "descriptive copy" by @adamhenriques for the line under continuos deliver. 
• MAIN EVENTS, you can order these as you wish @lakruzz maybe your best event or one you want to drive ticket sales biggest, then another 3, possibly up n coming the soonest. 
• SIDE BAR. min list of next two months. or enough to fill say may only has one then you'll have space for MAY, JUNE & July.  For exe. CODE CONFERENCE STOCKHOLM (no abbreviation) and date.
![codeconf_jc_home_v4](https://cloud.githubusercontent.com/assets/17589853/14815713/e41e310c-0ba4-11e6-919a-2a6ff231506c.jpg)

• GET INVOLVED. a brief rundown on what the different "events" are that you run. 
• 3 options bottom, sponsor inquires, contact us, lets be friends social link. 

finally the bottom bar with the words as is. (if spelt correctly) @lakruzz said we could take off devops. 

***images may change, for design/ layout only***

**Comment by jonnycoull** [comment]
![web layout](https://cloud.githubusercontent.com/assets/17589853/14816112/ffca7bca-0ba6-11e6-90bc-2fbe6a2e1dc7.jpg)

**Comment by lakruzz** [comment]
@jonnycoull  and @illus0r This case is a briefing case, we've been discussing design and layout back and forth. I Talked to @jonnycoull and it seems like he has a plan that we can carry out. 

@illus0r can you create the necessary workable sub tasks (Same prio as this on - give each task you create an estimate). Put all the tasks in this milestone ("Press ready" )

**Comment by jonnycoull** [comment]
@illus0r for the latest HEADER & FOOTER look at issue #135 before you race ahead on this.

**Comment by illus0r** [comment]
@jonnycoull Good!

**Comment by illus0r** [comment]
@timea done with subtasks
didn't make tasks for new menu items ![image](https://cloud.githubusercontent.com/assets/797993/15248390/a790ee92-1922-11e6-8421-0518c5669048.png) and corresponding pages. That's not about the frontpage.

**Comment by lakruzz** [comment]
I'm closing this one then, no more work is needed on the briefing now - that tasks we spun off it takes presendence
